### ISSUE TITLE
[Feature Request] Client-side rendering

### ISSUE THREAD

**Issue by Zulko** [ismay/coarse#16]
It seems that while the API allows only files for input and output, the code could also take a DOM svg element as an input, which would allow to create rough SVGs directly on the client side. Would that work ?

One advantage would be to allow the use of CSS classes to style SVGs (typically, add a color theme). (this would certainly require the users to first inline the CSS "computed" styles in the SVG elements, before running ``coarse``).

**Comment by Zulko** [comment]
Indeed. I was a bit short-sighted on this one. Maybe replace "server-side" by "client- or server-side" in the Github description ?

**Comment by ismay** [comment]
> Maybe replace "server-side" by "client- or server-side" in the Github description ?

Yeah definitely. I would want to address the jsdom dependency though. Wouldn't want jsdom to get bundled in a client-side app :)
